pkgs/development/libraries/capstone/default.nix

```diff
@@ -7,15 +7,22 @@
 
 stdenv.mkDerivation rec {
   pname = "capstone";
-  version = "4.0.2";
+  version = "5.0-post1";
 
   src = fetchFromGitHub {
     owner = "aquynh";
     repo = "capstone";
     rev = version;
-    sha256 = "sha256-XMwQ7UaPC8YYu4yxsE4bbR3leYPfBHu5iixSLz05r3g=";
+    sha256 = "sha256-KaQaUfsKUXxhx863zcq+y/2zl5sPa8iar9/2tdRiDJg=";
   };
 
+  # Fix improper path joining https://github.com/NixOS/nixpkgs/issues/144170
```

Comment:
This isn't enough. Please use https://github.com/capstone-engine/capstone/commit/b0d1725379a65b31fb5f637f81f1c12a1a2e840c.patch